ArduCopter/AP_Arming.cpp

```diff
@@ -332,7 +332,7 @@ bool AP_Arming_Copter::rc_calibration_checks(bool display_failure)
         copter.channel_yaw
     };
 
-    copter.ap.pre_arm_rc_check = rc_checks_copter_sub(display_failure, channels)
+    copter.ap.pre_arm_rc_check = uint8_t(rc_checks_copter_sub(display_failure, channels))
```

Comment:
Same here, sadly.

I don't suppose we could push a pragma that's specific enough to squelch the short-cutting warning?

Reply:
Fixed